lib/main.dart

```diff
@@ -93,78 +68,10 @@ class MyApp extends StatelessWidget {
-          ),
-          colorScheme: ColorScheme.fromSwatch().copyWith(
-            secondary: Asset.Colors.greenBlue,
-          ),
-        ),
-        darkTheme: ThemeData(
-          brightness: Brightness.dark,
-          hintColor: Asset.Colors.warmGray,
-          primaryColor: Asset.Colors.main,
-          primaryColorLight: const Color(0xff6d7999),
-          primaryColorDark: const Color(0xff172540),
-          secondaryHeaderColor: Asset.Colors.mediumGray,
-          backgroundColor: Asset.Colors.darkDim,
-          bottomAppBarColor: Asset.Colors.darkDim,
-          disabledColor: Asset.Colors.warmGray,
-          dialogBackgroundColor: Asset.Colors.dark,
-          textTheme: TextTheme(
-            headline1: TextStyle(color: Asset.Colors.light),
-            headline2: TextStyle(color: Asset.Colors.paleGray),
-            headline3: TextStyle(color: Asset.Colors.mediumGray),
-            caption: TextStyle(color: Asset.Colors.dark),
-          ),
-          colorScheme: ColorScheme.fromSwatch().copyWith(
-            secondary: Asset.Colors.greenBlue,
-            brightness: Brightness.dark,
-          ),
-        ),
-        routes: {
-          AuthSwitch.name: (BuildContext context) => AuthSwitch(),
-          Splash.name: (BuildContext context) => Splash(),
-          Tutorial.name: (BuildContext context) => Tutorial(),
-          Intro.name: (BuildContext context) => Intro(),
-          SignIn.name: (BuildContext context) => SignIn(),
-          SignUp.name: (BuildContext context) => SignUp(),
-          FindPw.name: (BuildContext context) => FindPw(),
-          MainEmpty.name: (BuildContext context) => MainEmpty(),
-          HomeTab.name: (BuildContext context) => HomeTab(),
-          Ledgers.name: (BuildContext context) => Ledgers(),
-          LedgerEdit.name: (BuildContext context) => LedgerEdit(),
-          LedgerView.name: (BuildContext context) => LedgerView(),
-          Terms.name: (BuildContext context) => Terms(),
-          ProfileMy.name: (BuildContext context) => ProfileMy(),
-          Setting.name: (BuildContext context) => Setting(),
-          SettingAnnouncement.name: (BuildContext context) =>
-              SettingAnnouncement(),
-          SettingOpinion.name: (BuildContext context) => SettingOpinion(),
-          SettingFAQ.name: (BuildContext context) => SettingFAQ(),
-          SettingNotification.name: (BuildContext context) =>
-              SettingNotification(),
-          LedgerItemEdit.name: (BuildContext context) => LedgerItemEdit(),
-          SettingCurrency.name: (BuildContext context) => SettingCurrency(),
-          SettingExcel.name: (BuildContext context) => SettingExcel(),
-          LockRegister.name: (BuildContext context) => LockRegister(),
-          LockAuth.name: (BuildContext context) => LockAuth(),
-          LineGraph.name: (BuildContext context) => LineGraph(),
-        },
+        theme: Themes.light,
+        darkTheme: Themes.dark,
+        themeMode: ThemeMode.light,
```

Comment:
It'd be better not to declare this.
Please let the app follows the device's brightness if nothing has been chosen yet.